index.js

```diff
@@ -97,12 +98,22 @@ module.exports._getDependencies = function(config) {
     return [];
   }
 
+
+  const tsMatchPath = (() => {
+    if (config.tsConfig) {
+      const absoluteBaseUrl = path.join(path.dirname(config.tsConfigPath), config.tsConfig.compilerOptions.baseUrl)
+      // REF: https://github.com/dividab/tsconfig-paths#creatematchpath
+      return createMatchPath(absoluteBaseUrl, config.tsConfig.compilerOptions.paths)
+    }
+    return (alias) => undefined;
+  })()
```

Comment:
@mrjoelkemp That is to prevent using `let` over `const`.

For example, let's say we should initialize `foo` by some condition.

```ts
// In this example, assume a ternary operator cannot be used, 
// as if-else blocks can be large in real cases.
let foo
  if (bar) {
    foo = 1
  } else {
    foo = 2
  }
```

However, `foo` is declared with `let`, so it can be mutated from somewhere else when it should not be.
To prevent this, some (especially functional) languages provides syntax sugar or special approach. 
For instance, Rust can treat `if-else` as an "expression", not a "statement".

```rs
// In Rust, pure `let` without `mut` keyword is equivalent to `const` in Javascript.
let foo = if bar {
    1
} else {
    2
};
```

In Javascript, calling IIFE for variable initialization is a similar pattern.

```ts
// foo now becomes `const` !
const foo = ((): void => {
    if (bar) {
      return 1
    }
    return 2
  })()
```